packages/core/client/src/schema-component/antd/association-field/AssociationFieldProvider.tsx

```diff
@@ -45,12 +48,53 @@ export const AssociationFieldProvider = observer(
 
     const [loading, setLoading] = useState(!field.readPretty);
 
+    const { loading: rLoading, run } = useRequest(
+      () => {
+        const targetKey = collectionField.targetKey;
+        if (!fieldSchema.default) {
+          return Promise.reject(null);
+        }
+        if (!['Picker', 'Select'].includes(currentMode)) {
+          return Promise.reject(null);
+        }
+        if (!_.isObject(fieldSchema.default)) {
+          return Promise.reject(null);
+        }
+        const ids = Array.isArray(fieldSchema.default)
+          ? fieldSchema.default.map((item) => item[targetKey])
+          : fieldSchema.default[targetKey];
+        if (!ids) {
+          return Promise.reject(null);
+        }
+        return api.request({
+          resource: collectionField.target,
+          action: Array.isArray(ids) ? 'list' : 'get',
+          params: {
+            filter: {
+              [targetKey]: ids,
+            },
+          },
+        });
+      },
+      {
+        manual: true,
+        onSuccess(res) {
+          field.initialValue = res?.data?.data;
+          field.value = res?.data?.data;
```

Comment:
```suggestion
          field.initialValue = res?.data?.data;
```

因为这是默认值，所以可以把 `field.value = res?.data?.data;` 去掉